docs/code_overview/models/liveupdate.rst

```diff
@@ -0,0 +1,5 @@
+LiveUpdate
```

Comment:
Also: Feel free to squash this commit into the first one since I consider it to be part of the atomic commit.

We'll likely also need a changelog entry too.